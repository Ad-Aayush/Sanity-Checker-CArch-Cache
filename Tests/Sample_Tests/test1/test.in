load ./Tests/Sample_Tests/test1/input1.s
step
step
step
step
mem 0x10000 4
regs 
run
step
exit