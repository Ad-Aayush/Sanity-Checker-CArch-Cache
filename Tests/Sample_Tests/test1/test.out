Executed: addi x5, x5, 0; PC=0x00000000
Executed: lui x4, 0x10; PC=0x00000004
Executed: addi x5, x5, -1; PC=0x00000008
Executed: sd x5, 2(x4); PC=0x0000000c
Memory[0x10000] = 0x0
Memory[0x10001] = 0x0
Memory[0x10002] = 0xFF
Memory[0x10003] = 0xFF
Registers:
x0  = 0x0
x1  = 0x0
x2  = 0x0
x3  = 0x0
x4  = 0x10000
x5  = 0xFFFFFFFFFFFFFFFF
x6  = 0x0
x7  = 0x0
x8  = 0x0
x9  = 0x0
x10 = 0x0
x11 = 0x0
x12 = 0x0
x13 = 0x0
x14 = 0x0
x15 = 0x0
x16 = 0x0
x17 = 0x0
x18 = 0x0
x19 = 0x0
x20 = 0x0
x21 = 0x0
x22 = 0x0
x23 = 0x0
x24 = 0x0
x25 = 0x0
x26 = 0x0
x27 = 0x0
x28 = 0x0
x29 = 0x0
x30 = 0x0
x31 = 0x0
Executed addi x5, x5, 1; PC=0x00000010
Nothing to step
Exited the simulator
